configuration.packages.json

```diff
@@ -72,22 +72,22 @@
     },
     "EdFi.Database.Admin.PostgreSQL": {
       "PackageName": "EdFi.Database.Admin.PostgreSQL",
-      "PackageVersion": "6.2.469",
+      "PackageVersion": "6.2.536",
```

Comment:
Does this package need an update? This is the database backup file used in the docker deployments.